### ISSUE TITLE
Cannot send a message in Google Hangouts

### ISSUE THREAD

**Issue by igorlash** [brave/browser-laptop#10862]
I cannot send a message in Google Hangouts after the last update. 

- Platform : Win10

- Brave Version (revision SHA):  0.18.29

- Steps to reproduce:
    1. Open any google email account
    2. Open a Google Hangout window
    3. Try to send a message

    
- Actual result:
  Shows information that the message cannot be sent

- Expected result:
 The message should be sent. 

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
This problem started to reproduce after the last update 

- Is this an issue in the currently released version?
 Yes. 

- Can this issue be consistently reproduced?

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:

- Any related issues:

**Comment by srirambv** [comment]
Works for me. What shields settings are you using

**Comment by igorlash** [comment]
My shield settings is 
![shield_settings](https://user-images.githubusercontent.com/31065075/30205163-b4ec7cec-9490-11e7-8b81-c55010d54511.png)

With disabled shield settings messages can be sent.

**Comment by srirambv** [comment]
You need to set Allow all cookies to work.

**Comment by igorlash** [comment]
Yes, now it works. 
But it wasn't obvious after the last update. It just stopped working.

**Comment by tbass71** [comment]
Yes, I found too that it works if you shut the shields off...more specifically the cookies. This is more than likely a change by Google and not the fault of Brave. There are many sites that are circumventing tracking protection. These browser developers can only do so much.

**Comment by bsclifton** [comment]
gchat issue report: https://github.com/brave/browser-laptop/issues/10875

As noted, if you allow 3rd party cookies it will work as expected. We have some exceptions which you can see here:
https://github.com/brave/browser-laptop/blob/master/js/data/siteHacks.js

**Comment by qippur** [comment]
Great! 
Forgive a noob question: should we download siteHacks.js? Where is it supposed to be placed?
Regards

--
